server/src/main/scala/net/wiringbits/repositories/daos/UserNotificationsDAO.scala

```diff
@@ -0,0 +1,64 @@
+package net.wiringbits.repositories.daos
+
+import net.wiringbits.repositories.models.{NotificationStatus, UserNotification}
+
+import java.sql.Connection
+import java.time.Instant
+import java.util.UUID
+
+object UserNotificationsDAO {
+
+  import anorm._
+
+  def create(request: UserNotification.Create)(implicit conn: Connection): Unit = {
+    val _ = SQL"""
+      INSERT INTO user_notifications
+        (user_notification_id, user_id, notification_type, subject, message, status, execute_at)
```

Comment:
Let's include `created_at`/`updated_at`, I'm trying to avoid using timestamps from SQL because it complicates writing tests